Doc/reference/expressions.rst

```diff
@@ -1388,8 +1395,11 @@ The shifting operations have lower priority than the arithmetic operations:
 These operators accept integers as arguments.  They shift the first argument to
 the left or right by the number of bits given by the second argument.
 
-This operation can be customized using the special :meth:`~object.__lshift__` and
-:meth:`~object.__rshift__` methods.
+The left shift operation can be customized using the special :meth:`~object.__lshift__`
+and :meth:`~object.__rlshift__` methods.
+
```

Comment:
Would not be better to merge these two paragraphs?
```suggestion
```

Reply:
For sure, merged it and did the same formatting for division and floor division operations. 👍